src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_enkf_update.F90

```diff
@@ -2100,14 +2099,15 @@ subroutine write_smapL4SMaup( option, date_time, work_path, exp_id, N_ens,    &
 
     character(len=*), parameter :: Iam = 'write_smapL4SMaup'
     character(len=400) :: err_msg
+    character(len=10)  :: gridname_tmp
 
     ! --------------------------------------------------------------
     !
     ! smapL4SMaup output only works for 9 km EASE grids
 
-    if ( (trim(tile_grid_g%gridtype)/='EASE_M09'  )  .and.                  &
+    if ( (trim(tile_grid_g%gridtype)/='EASEv1_M09'  )  .and.                  &
          (trim(tile_grid_g%gridtype)/='EASEv2_M09')         )               then
-       err_msg = 'out_smapL4SMaup requires tile-space for 9 km EASE[v2] grid'
+       err_msg = 'out_smapL4SMaup requires tile-space for 9 km EASEv1 or EASEv2 grid'
```

Comment:
@weiyuan-jiang : I tried the nightly tests on the latest branch (https://github.com/GEOS-ESM/GEOSldas/pull/586/commits/5bd729d90ea0bca01bcd1665690e584366f8033e).  
The LDAS_GLOBAL/assim test fails with the following error message, which comes from this if block:
"LDAS ERROR (3000) from write_smapL4SMaup: out_smapL4SMaup requires tile-space for 9 km EASEv1 or EASEv2 grid"
Did the value of `tile_grid_g%gridtype` at this point in the test case change?  Could this be "EASEv2_M09" vs just "M09"?  See also line 2232 below.

Reply:
The problem is in LDAS.rc with grid name as SMAP-EASEv2-M09. We either change it to SMAP_EASEv2_M09 or change the code here if (index(...., 'M09') == 0) then

Reply:
For backward compatibility with existing LDAS.rc files, let's use  `if (index(...., 'M09') == 0)`.  It's extremely unlikely that there'll ever be another, non-EASE gridtype that has the string "M09" in the name.

Reply:
sure. changed. Hopefully the nighty test will pass this point.

Reply:
@weiyuan-jiang, I retested the updated branch.  Unfortunately, I'm still getting a runtime error in the LDAS_GLOBAL/assim test.  GEOSldas exits with the following message:
` LDAS ERROR (3000) from scale_obs_Tb_zscore: tile_id not found in scaling parameter file`
This doesn't make any sense to me.  The tile IDs and scaling parameters should be unchanged in the inputs from the test.  Did we touch the tile IDs anywhere in the branch?